sdk/iothub/arm-iothub/samples/v6/javascript/iotHubResourceCreateOrUpdate.js

```diff
@@ -34,7 +34,7 @@ async function iotHubResourceCreateOrUpdate() {
         },
         maxDeliveryCount: 10,
       },
-      enableDataResidency: true,
+      enableDataResidency: false,
```

Comment:
out of curiosity, what does this do?

Reply:
@xirzec https://github.com/Azure/azure-sdk-for-js/pull/22063#issuecomment-1147117291 do you think we should fix it here ? Or should we file a PR in the rest api specs and then refresh it with latest rest api specs ?

Reply:
@qiaozha yes, excellent point, we should fix this upstream as well. I don't mind if we'd rather fix it in both places than have to refresh, but if it's quick and easy to refresh, I'm good taking that approach.

Reply:
Actually, it's not that eary to refresh, I think we can merge this PR first and then I will file a PR in rest api specs to fix those examples. Thanks

Reply:
@qiaozha Okay. I agree your advice. If you have no question about this PR, please help approve it. Thanks a lot.